src/facade/reply_builder.h

```diff
@@ -155,6 +155,27 @@ class SinkReplyBuilder {
 
   virtual size_t UsedMemory() const;
 
+  enum SendStatsType {
+    kRegular,
+    kBatch,
+    kCount,
```

Comment:
can you add a comment about each of them? 
I first thought `kCount` is another type, maybe `kNumTypes` ?

Reply:
done